packages/text-input/src/TextInput.spec.tsx

```diff
@@ -222,6 +222,21 @@ describe('packages/text-input', () => {
     });
   });
 
+  describe('when the "sizeVariant" is "large"', () => {
+    test('check if font-size is 18px', () => {
+      const { label } = renderTextInput({
+        value: validEmail,
+        sizeVariant: SizeVariant.Large,
+        optional: true,
+        ...defaultProps,
+      });
+
+      expect(label).toHaveStyle({
+        fontSize: '18px',
+      });
+    });
+  });
+
```

Comment:
This works for now, but this type of test is better suited to something like Chromatic or Percy

Reply:
We could even consider using this package https://www.npmjs.com/package/jest-image-snapshot